### PR TITLE
Add a stub page for upgrading to Rails 5

### PR DESCRIPTION
**Pull request by boffbowsh**

### CODE REVIEW

File: source/manual/upgrading-rails.html.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-10-28
+title: Upgrading Rails
+section: Packaging
+layout: manual_layout
+parent: "/manual.html"
+---
+
+When upgrading our apps between Rails minor versions, follow the [official Rails guides][guide]
```
Comment (andrewgarner) [inline source/manual/upgrading-rails.html.md]: ~~minor~~

Comment (boffbowsh) [inline source/manual/upgrading-rails.html.md]: It's still useful for upgrading from e.g. 5.0 to 5.1?

File: source/manual/upgrading-rails.html.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-10-28
+title: Upgrading Rails
```
Comment (tijmenb) [inline source/manual/upgrading-rails.html.md]: Following the [styleguide for titles](https://docs.publishing.service.gov.uk/manual/create-page.html#title) this should probably be something like "Upgrade Rails"

File: source/manual/upgrading-rails.html.md (hunk 1 above)
Comment (andrewgarner) [inline source/manual/upgrading-rails.html.md]: It's for major and minor upgrades.

File: source/manual/upgrade-rails.html.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-10-28
```
Comment (tijmenb) [inline source/manual/upgrade-rails.html.md]: This needs to be changed after https://github.com/alphagov/govuk-developer-docs/pull/185.

```
last_reviewed_at: 2017-04-28
review_in: 6 months
```

**Comment by boffbowsh** [comment]
Done done and done.

**Pull request merged**